security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,21 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-3-0"]
+== OpenShift Compliance Operator 1.3.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The DISA-STIG profile is now available for Compliance Operator 1.3.0. See xref:../../security/compliance_operator/compliance-operator-supported-profiles.adoc#compliance-supported-profiles_compliance-operator-supported-profiles[Supported compliance profiles]
```

Comment:
I would update "DISA-STIG profile" with the full name: 

"The Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG) for {product-title} is now available..." 

Additionally, we need a period after the xref sentence. I would also add something like:

"For more information, see xref:..[words]." 

Or: 
"See xref:..[words] for additional information."

Reply:
Done, thank you!